nav2_behaviors/plugins/assisted_teleop.cpp

```diff
@@ -70,16 +59,25 @@ Status AssistedTeleop::onRun(const std::shared_ptr<const AssistedTeleopAction::G
     std::bind(
       &AssistedTeleop::joyCallback,
       this, std::placeholders::_1));
+}
+
+Status AssistedTeleop::onRun(const std::shared_ptr<const AssistedTeleopAction::Goal> command)
+{
+  command_time_allowance_ = command->time_allowance;
+  end_time_ = steady_clock_.now() + command_time_allowance_;
 
+  auto node = node_.lock();
+  if (!node) {
```

Comment:
You don't need any of this node lock stuff

Reply:
Okay. It is also in backup and spin.

Reply:
But you don't use it in this function, they use it in those functions, no?

Reply:
Yep. Removed.